tracer/src/Datadog.Trace/SpanContext.cs

```diff
@@ -70,6 +71,7 @@ public SpanContext(ulong? traceId, ulong spanId, SamplingPriority? samplingPrior
             // - samplingPriority: SamplingPriority? => int?
             SpanId = spanId;
             SamplingPriority = (int?)samplingPriority;
+            IsRemote = true;
```

Comment:
This doesn't _have_ to have been propagated by a remote parent 🤔 The customer can create a `SpanContext` with this API and it could represent a "remote" parent, or it could be from a different local process etc. I'm not sure if that's an issue or not. Or if it means `IsRemote` should have a slightly different name 🤷‍♂️

Reply:
Hmm yeah I was wondering about that, let me think about that - I may change where it gets set to avoid that case